### ISSUE TITLE
External dependencies

### ISSUE THREAD

**Issue by theChaosCoder** [vapoursynth/vsrepo#14]
There are some plugins like https://github.com/WolframRhodium/VapourSynth-BilateralGPU or waifu2x-caffe that requieres gpu runtimes like cuda, opencl etc. 

I don't think that vsrepo should handle these dependencies, but instead show a POST MSG after installing the plugin that this plugin has further dependencies. Something like that.

What do you think?

**Comment by myrsloik** [comment]
OpenCL just requires a working GPU and driver. It's not a real dependency. A windows 7 or later system with non-ancient hardware and drivers is always assumed and always will be. The plugin itself will return the error if necessary.

CUDA is more interesting since it's restricted to certain hardware. Unfortunately hardware detection is a shitshow in hybrid graphics solutions so it's better to simply install it too.

Basically what I'm trying to say is that hardware requirements are outside the scope of this project. It only handles the userspace software.
